firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/CrashlyticsRegistrar.java

```diff
@@ -24,20 +24,29 @@
 import com.google.firebase.inject.Deferred;
 import com.google.firebase.installations.FirebaseInstallationsApi;
 import com.google.firebase.platforminfo.LibraryVersionComponent;
+import com.google.firebase.sessions.FirebaseSessions;
+import com.google.firebase.sessions.api.FirebaseSessionsDependencies;
+import com.google.firebase.sessions.api.SessionSubscriber;
 import java.util.Arrays;
 import java.util.List;
 
 /** @hide */
 public class CrashlyticsRegistrar implements ComponentRegistrar {
   private static final String LIBRARY_NAME = "fire-cls";
 
+  static {
+    // Add Crashlytics as a dependency of Sessions when this class is loaded into memory.
+    FirebaseSessionsDependencies.INSTANCE.addDependency(SessionSubscriber.Name.CRASHLYTICS);
+  }
+
   @Override
   public List<Component<?>> getComponents() {
     return Arrays.asList(
         Component.builder(FirebaseCrashlytics.class)
             .name(LIBRARY_NAME)
             .add(Dependency.required(FirebaseApp.class))
             .add(Dependency.required(FirebaseInstallationsApi.class))
+            .add(Dependency.required(FirebaseSessions.class))
```

Comment:
Just a curiosity question here: Is the order important here? If so, why are we above the NativeComponent?

Reply:
I don't think the order is important. I just grouped it with other Firebase* deps. Let me know if the order does matter and I can move it. But looking at the other deps, I don't see any real order.